source/vibe/inet/path.d

```diff
@@ -94,17 +94,17 @@ struct Path {
 	}
 	
 	/// Converts the Path object to a native path string (backslash as path separator on Windows).
-	string toNativeString()
+	string toNativeString() nothrow
 	const {
 		Appender!string ret;
 		
 		// for absolute unix paths start with /
-		version(Posix) { if(absolute) ret.put('/'); }
+		version(Posix) { if (m_absolute) ret.put('/'); }
 		
 		foreach( i, f; m_nodes ){
 			version(Windows) { if( i > 0 ) ret.put('\\'); }
 			version(Posix) { if( i > 0 ) ret.put('/'); }
-			else { enforce("Unsupported OS"); }
+			else { static assert(0, "Unsupported OS"); }
```

Comment:
Looks like this will fail compilation on Windows. Either `else version(Posix)` or replacing the enforce with an `assert` should be ok. Not sure why I went with a runtime error at the time...